requirements.txt

```diff
@@ -1,2 +1,3 @@
 pymc>=5.8.2
+formulae
```

Comment:
Make installation optional?

Reply:
How we structure the optional dependencies in pip?

Reply:
With setup.py? https://github.com/pymc-devs/pymc-experimental/blob/79c4dc13e9ea9a5cd6af6bc76f70f7b0c948af78/setup.py#L60-L65

Reply:
yep, I meant every time there is a new optional dependency, there is a new name here, easy to get lost

Reply:
Put it in `complete` only. I think we only need two. Otherwise people can manually install, because when they try to use a function we give them an informative message they need to install x